### DISCUSSION TITLE
Uploading more than one document in hosting requirement

### DISCUSSION THREAD

**Discussion by AGNMM18** [CareTogether/CareTogetherCMS#496] [Ideas]
Can we add the ability to upload more than one document if needed in any requirement function - volunteer or referral?

**Comment by LarsKemmann** [comment]
Currently, the system isn't designed to _link_ the requirement completion to more than one document at a time (mostly because of wanting to preserve a clear audit link between a document and the requirement(s) it is linked to). But maybe there is a use case for this?
